tests/stdlib/thttpclient.nim

```diff
@@ -8,7 +8,7 @@ discard """
 import strutils
```

Comment:
any way to remove `disabled: true` or use `disabled: <insert bad os>` ? your code won't run in CI currently

Reply:
=> I'm attempting that here: https://github.com/nim-lang/Nim/pull/16710